inst/include/RcppArmadilloAs.h

```diff
@@ -32,10 +32,17 @@ namespace traits {
         Exporter(SEXP x) : data(x){}
 
         inline arma::field<T> get() {
-            size_t n = data.size() ;
```

Comment:
This does only extracts length. We want to access `.attr("dim"); `here to retain the object dimensions.